components/InlineDropDownMenu/InlineDropDownMenuItem.js

```diff
@@ -0,0 +1,85 @@
+import React, { PureComponent } from 'react';
+import autoBindReact from 'auto-bind/react';
+import { Animated, Dimensions } from 'react-native';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import { connectStyle } from '@shoutem/theme';
+import { TouchableOpacity } from '../TouchableOpacity';
+import { Text } from '../Text';
+
+const window = Dimensions.get('window');
+const AnimatedTouchable = Animated.createAnimatedComponent(TouchableOpacity);
+
+class InlineDropDownMenuItem extends PureComponent {
+  static propTypes = {
+    item: PropTypes.object,
+    index: PropTypes.number,
+    selectedDescriptor: PropTypes.string,
+    onItemPressed: PropTypes.func,
+    isSelected: PropTypes.bool,
+  };
+
+  constructor(props) {
+    super(props);
+
+    autoBindReact(this);
+
+    this.animatedValue = new Animated.Value(0);
+  }
+
+  componentDidMount() {
+    const { index } = this.props;
+
+    Animated.timing(
+      this.animatedValue,
+      {
+        toValue: 1,
+        useNativeDriver: true,
+        duration: 300 + index * 15,
+      }
+    ).start();
+  }
+
+  handlePress() {
+    const { onItemPressed, item } = this.props;
+
+    if (onItemPressed) {
+      onItemPressed(item);
+    }
+  }
+
+  render() {
+    const { isSelected, selectedDescriptor, item, style } = this.props;
+
+    const resolvedText = isSelected ? `${item.title} (${selectedDescriptor})` : item.title;
+    const textStyle = isSelected ? 'muted' : '';
+
+    return (
+      <AnimatedTouchable
+        style={[
+          style.container,
+          {
+            transform: [
+              {
+                translateX: this.animatedValue.interpolate({
+                  inputRange: [0, 1],
+                  outputRange: [window.width, 0]
```

Comment:
Trailing commas? I assume prettier was just doing what the config from @shoutem/ui was telling it to, should we look into that?

Reply:
I'm all up for copying lint configs to ui ( that we have in mobile repo ) if we don't have them already